sklearn/multiclass.py

```diff
@@ -513,6 +514,11 @@ def partial_fit(self, X, y, classes=None):
                                 range(self.n_classes_ *
                                 (self.n_classes_-1) // 2)]
 
+        if not set(self.classes_).issuperset(y):
```

Comment:
If you are using `np.unique(y)` below, you might as well store `np.unique(y)` and raise an Error `if not all(np.in1d(np.unique(y), self.classes_)`